packages/odyssey-react/src/components/TagList/TagList.theme.ts

```diff
@@ -13,19 +13,14 @@
 import type { ThemeReducer } from "@okta/odyssey-react-theme";
 
 export const theme: ThemeReducer = (theme) => ({
-  // Color
-  BackgroundColorItem: theme.ColorPaletteNeutral100,
-
-  // Radius
-  BorderRadiusItem: theme.BorderRadiusBase,
-
-  // Space
-  SpaceMarginBlockEnd: theme.SpaceRemXs,
-  SpaceMarginInlineStart: theme.SpaceRemXs,
-  SpaceMarginBlockStartItem: 0,
-  SpaceMarginBlockEndItem: theme.SpaceRemXs,
-  SpaceMarginInlineStartItem: 0,
-  SpaceMarginInlineEndItem: theme.SpaceRemXs,
-  SpacePaddingBlockItem: 0,
-  SpacePaddingInlineItem: theme.SpaceEmXs,
+  ItemBackgroundColor: theme.ColorPaletteNeutral100,
```

Comment:
is there no approved/functional background color that works for this yet?

Reply:
There is not